README.md

```diff
@@ -509,7 +618,10 @@ def resource_validation_run_batch(context, data_dict):
 
 ### Starting the validation process manually
 
-You can start (asynchronous) validation jobs from the command line using the `paster validation run` command. If no parameters are provided it will start a validation job for all resources in the site of suitable format (ie `ckanext.validation.formats`):
+You can start (asynchronous) validation jobs from the command line using the
+`paster validation run` command. If no parameters are provided it will start a
```

Comment:
Looks like we forgot to migrate this command and the `report `and `report-full` ones to Click :scream: 
We can do it in a separate PR